app/views/reports/regions/_treatment_outcomes_card.html.erb

```diff
@@ -7,7 +7,7 @@
   </h3>
 
   <div class="d-flex fd-column fd-lg-row">
-      <div class="pl-lg-12px w-lg-50 h-lg-auto h-print-12cm">
+      <div class="pl-lg-12px w-lg-50 h-lg-auto h-print-12cm" style="min-height: 300px;">
```

Comment:
I know it's annoying 😅 but can we introduce this CSS through a class and a rule in the appropriate CSS file? Let's avoid inline styles.

Reply:
I was wondering what's the point on this other than philosophy? Now we're writing in very specific rules into our whole CSS file for 1-off formatting on a single page.

Reply:
Anyhow, I'll move it.

Reply:
The philosophy is consistency. We want to minimize the number of places we need to look to find out where an element's style is coming from.